packages/desktop-client/src/components/budget/index.tsx

```diff
@@ -313,7 +313,25 @@ function BudgetInner(props: BudgetProps) {
     }
   };
 
+  const groupNameAlreadyExistsNotification = name => {
+    props.addNotification({
+      type: 'error',
+      message: `Group ‘${name}’ already exists in budget (May be Hidden)`,
```

Comment:
I think you can pass in the existing group in this method and set the message accordingly if the group is actually hidden.